From 4ad1abb2b2827d8a54f85413bba9aa0d2b5da727 Mon Sep 17 00:00:00 2001
From: Nick Larsen <nick.larsen@stackable.tech>
Date: Thu, 15 Oct 2026 13:36:40 +0200
Subject: Remove Develocity Maven extension

The pinned develocity-maven-extension 2.1 injects Maven's internal
PluginDescriptorCache, which was reworked in Maven 3.10.0, so every mvn
invocation aborts in afterProjectsRead with a Guice provisioning error.

We do not publish build scans and cannot authenticate against
develocity.apache.org anyway, so drop the extension and its config
instead of bumping it.
---
 .mvn/develocity.xml | 47 ---------------------------------------------
 .mvn/extensions.xml | 34 --------------------------------
 2 files changed, 81 deletions(-)
 delete mode 100644 .mvn/develocity.xml
 delete mode 100644 .mvn/extensions.xml

diff --git a/.mvn/develocity.xml b/.mvn/develocity.xml
deleted file mode 100644
index f320f96953..0000000000
--- a/.mvn/develocity.xml
+++ /dev/null
@@ -1,47 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
-<!--
-
-    Licensed to the Apache Software Foundation (ASF) under one
-    or more contributor license agreements.  See the NOTICE file
-    distributed with this work for additional information
-    regarding copyright ownership.  The ASF licenses this file
-    to you under the Apache License, Version 2.0 (the
-    "License"); you may not use this file except in compliance
-    with the License.  You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing,
-    software distributed under the License is distributed on an
-    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-    KIND, either express or implied.  See the License for the
-    specific language governing permissions and limitations
-    under the License.
-
--->
-<develocity
-        xmlns="https://www.gradle.com/develocity-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="https://www.gradle.com/develocity-maven https://www.gradle.com/schema/develocity-maven.xsd">
-    <projectId>nifi</projectId>
-    <server>
-        <url>https://develocity.apache.org</url>
-    </server>
-    <buildScan>
-        <backgroundBuildScanUpload>#{isFalse(env['GITHUB_ACTIONS'])}</backgroundBuildScanUpload>
-        <publishing>
-            <onlyIf>authenticated</onlyIf>
-        </publishing>
-        <obfuscation>
-            <ipAddresses>#{{'0.0.0.0'}}</ipAddresses>
-        </obfuscation>
-    </buildScan>
-    <buildCache>
-        <local>
-            <enabled>#{isFalse(env['GITHUB_ACTIONS'])}</enabled>
-        </local>
-        <remote>
-            <enabled>true</enabled>
-            <storeEnabled>#{isTrue(env['GITHUB_ACTIONS']) and isTrue(env['DEVELOCITY_ACCESS_KEY'])}</storeEnabled>
-        </remote>
-    </buildCache>
-</develocity>
diff --git a/.mvn/extensions.xml b/.mvn/extensions.xml
deleted file mode 100644
index 3eddf27e9f..0000000000
--- a/.mvn/extensions.xml
+++ /dev/null
@@ -1,34 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
-<!--
-
-    Licensed to the Apache Software Foundation (ASF) under one
-    or more contributor license agreements.  See the NOTICE file
-    distributed with this work for additional information
-    regarding copyright ownership.  The ASF licenses this file
-    to you under the Apache License, Version 2.0 (the
-    "License"); you may not use this file except in compliance
-    with the License.  You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing,
-    software distributed under the License is distributed on an
-    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-    KIND, either express or implied.  See the License for the
-    specific language governing permissions and limitations
-    under the License.
-
--->
-<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
-    <extension>
-        <groupId>com.gradle</groupId>
-        <artifactId>develocity-maven-extension</artifactId>
-        <version>2.1</version>
-    </extension>
-    <extension>
-        <groupId>com.gradle</groupId>
-        <artifactId>common-custom-user-data-maven-extension</artifactId>
-        <version>2.0.5</version>
-    </extension>
-</extensions>
